### PR TITLE
chore(release): pull release/v1.86.0 into main

### PR DESCRIPTION
**Pull request by devops-github-rudderstack**
:crown: *An automated PR*

### CODE REVIEW

**Review by achettyiitr — DISMISSED** [review]

File: src/v0/destinations/braze/util.js [hunk 1]
```diff
@@ -490,6 +540,7 @@ const processBatch = (transformedEvents) => {
   prepareGroupAndAliasBatch(mergeUsersArrayChunks, responseArray, destination, 'merge');
 
   if (successMetadata.length > 0) {
+    console.log(`Response 1 batchRequest ${JSON.stringify(responseArray)}`);
```
Comment (krishna2020) [inline src/v0/destinations/braze/util.js:543]: @aashishmalik remove the log here

File: src/v0/destinations/google_adwords_offline_conversions/transform.js [hunk 2]
```diff
@@ -48,6 +49,9 @@ const getConversions = (message, metadata, { Config }, event, conversionType) =>
       filteredCustomerId,
       eventLevelConsentsData,
     );
+    convertedPayload.payload.conversions[0] = updateConversion(
```
Comment (krishna2020) [inline src/v0/destinations/google_adwords_offline_conversions/transform.js:52]: Are no checks required here on non-empty conversions here ? And why handling only one conversion ? @ItsSudip

File: src/v0/destinations/braze/util.js (hunk 1 above)
Comment (aashishmalik) [reply src/v0/destinations/braze/util.js]: removed

**Review by krishna2020 — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

File: src/v0/destinations/google_adwords_offline_conversions/transform.js (hunk 2 above)
Comment (ItsSudip) [reply src/v0/destinations/google_adwords_offline_conversions/transform.js:52]: We don't need to check that. The way config json is written will always create one conversion.

Comment (krishna2020) [reply src/v0/destinations/google_adwords_offline_conversions/transform.js:52]: then why do we have it as an array ? legacy code ?

Comment (ItsSudip) [reply src/v0/destinations/google_adwords_offline_conversions/transform.js:52]: We need to send this in array format only. Conversions is an array of conversion objects.

Comment (krishna2020) [reply src/v0/destinations/google_adwords_offline_conversions/transform.js:52]: Then handling in as part of payload preparation itself would have sufficed - https://github.com/rudderlabs/rudder-transformer/blob/develop/src/v0/destinations/google_adwords_offline_conversions/utils.js#L385

Now the transformation logic is split in 2 places. Is updateConversion handles if Array.isArray(products) is false ?

Comment (ItsSudip) [reply src/v0/destinations/google_adwords_offline_conversions/transform.js:52]: No that is not required. That is already being handled before the `updateConversion` function call.

**Review by utsabc — APPROVED** [review]

**Review by ItsSudip — DISMISSED** [review]

**Review by shrouti1507 — DISMISSED** [review]

**Review by krishna2020 — APPROVED** [review]

**Review by utsabc — APPROVED** [review]

**Pull request merged**